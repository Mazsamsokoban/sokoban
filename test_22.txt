loadtest 22
showstate w
listboxes
move w right
showmaze
showstate w
listboxes