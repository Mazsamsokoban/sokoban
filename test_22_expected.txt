f0:w f1:box1 f2:box2 bf:null 

name:w
field:f0
direction:Neutral
points:0
pushforce:10.0

name:box1
field:f1
direction:Neutral
owner:null
pusher:null

name:box2
field:f2
direction:Neutral
owner:null
pusher:null

f0:null f1:w f2:box1 bf:box2 

name:w
field:f1
direction:Right
points:1
pushforce:10.0

name:box1
field:f2
direction:Right
owner:null
pusher:w

name:box2
field:bf
direction:Right
owner:w
pusher:w

